src/datasets.cpp

```diff
@@ -39,409 +39,352 @@
-        DataSet(IptcDataSets::Language, "Language", N_("Language"),
-                N_("Describes the major national language of the object, according "
-                "to the 2-letter codes of ISO 639:1988. Does not define or imply "
-                "any coded character set, but is used for internal routing, e.g. to "
-                "various editorial desks."),
-                false, false, 2, 3, Exiv2::string, IptcDataSets::application2, ""),
-        DataSet(IptcDataSets::AudioType, "AudioType", N_("Audio Type"),
-                N_("Indicates the type of an audio content."),
-                false, false, 2, 2, Exiv2::string, IptcDataSets::application2, ""),
-        DataSet(IptcDataSets::AudioRate, "AudioRate", N_("Audio Rate"),
-                N_("Indicates the sampling rate in Hertz of an audio content."),
-                false, false, 6, 6, Exiv2::string, IptcDataSets::application2, ""),
-        DataSet(IptcDataSets::AudioResolution, "AudioResolution", N_("Audio Resolution"),
-                N_("Indicates the sampling resolution of an audio content."),
-                false, false, 2, 2, Exiv2::string, IptcDataSets::application2, ""),
-        DataSet(IptcDataSets::AudioDuration, "AudioDuration", N_("Audio Duration"),
-                N_("Indicates the duration of an audio content."),
-                false, false, 6, 6, Exiv2::string, IptcDataSets::application2, ""),
-        DataSet(IptcDataSets::AudioOutcue, "AudioOutcue", N_("Audio Outcue"),
-                N_("Identifies the content of the end of an audio object data, "
-                "according to guidelines established by the provider."),
-                false, false, 0, 64, Exiv2::string, IptcDataSets::application2, ""),
-        DataSet(IptcDataSets::PreviewFormat, "PreviewFormat", N_("Preview Format"),
-                N_("A binary number representing the file format of the object data "
-                "preview. The file format must be registered with IPTC or NAA organizations "
-                "with a unique number assigned to it."),
-                false, false, 2, 2, Exiv2::unsignedShort, IptcDataSets::application2, ""),
-        DataSet(IptcDataSets::PreviewVersion, "PreviewVersion", N_("Preview Version"),
-                N_("A binary number representing the particular version of the "
-                "object data preview file format specified in tag <PreviewFormat>."),
-                false, false, 2, 2, Exiv2::unsignedShort, IptcDataSets::application2, ""),
-        DataSet(IptcDataSets::Preview, "Preview", N_("Preview Data"),
-                N_("Binary image preview data."),
-                false, false, 0, 256000, Exiv2::undefined, IptcDataSets::application2, ""),
-        DataSet(0xffff, "(Invalid)", N_("(Invalid)"),
-                N_("(Invalid)"),
-                false, false, 0, 0, Exiv2::unsignedShort, IptcDataSets::application2, "")
+    static constexpr DataSet application2Record[] = {
```

Comment:
Could you please revert the formatting changes here and just keep the 1-line change about `static constexpr` ?